core/debugger/engine_profiler.h

```diff
@@ -32,6 +32,7 @@
 
 #include "core/object/gdvirtual.gen.inc"
 #include "core/object/ref_counted.h"
+#include "core/object/script_language.h"
```

Comment:
There are a small number of cases where a header needs a class to work, but wasn't including it. These things were being included transitively when 3D is enabled in the editor, but fail to compile otherwise. This PR fixes these cases by adding the includes, including this one include in the core folder, and a few others elsewhere.